setup.py

```diff
@@ -40,6 +40,6 @@
     setup_requires=['wheel'],
     install_requires=[
         'Django>=2.2',
-        'WeasyPrint>=43',
+        'WeasyPrint==53',
```

Comment:
weasyprint **does not** follow semantic versioning https://doc.courtbouillon.org/weasyprint/latest/api_reference.html#versioning https://github.com/Kozea/WeasyPrint/issues/900

Please keep the >= version range specifier because another API change will probably not be seen in a while but pinning to ==53 (yes, pinning, because weasyprint only publishes major versions 53.X will never be published, next release will be 54[.0.0] ) would cause severe issues to end users when trying to resolve dependencies!

It is unfortunate for us that weasyprint API changes can't be avoided through version specifiers, but we will have to deal with it.

Reply:
Hi @n1ngu thank you for the feedback, I pushed the change